modules/2d_inference_3d_volume.ipynb

```diff
@@ -0,0 +1,217 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #46.                x = x.squeeze(dim=self.slice_axis + 2)
</pre><p>Maybe <code>self.slice_axis + 2</code> is not correct when <code>self.slice</code> is 2?</p><p> </p><p>Thanks.</p><p> </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Project-MONAI/tutorials/pull/479/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='5'/>

Reply:
It should still work when `self.slice_axis` is 2, this would mean it squeezes over the index 4 of the `[N,C,D,H,W]` tensor, which corresponds with `W`.

Reply:
Maybe I misunderstood the logic, I mean is there any case that `self.slice_axis + 2` out of index?

Thanks.

Reply:
Aah yes, I see.

So the `self.slice_axis` should only be used for axis 0, 1, 2 which corresponds to `[D, H, W]` respectively. I'll add a check to make this clear then!

Reply:
And I would suggest to use `spatial_dim` instead of `axis`, then users can easily understand the values.

Thanks.